### DISCUSSION TITLE
Continued support after technical workshops

### DISCUSSION THREAD

**Discussion by karenword** [carpentries/trainers#117] [Local Communities]
How can local communities support technical trainees in continuing to develop skills after a workshop?

**Comment by karenword** [comment]
This question was addressed at a Trainers meeting on June 17, 2021. This is a summary of meeting responses.

**Community-based support**
* optional follow-up sessions for content skipped in a workshop
* time after a workshop to practice
* regular meetups: study groups, hacky hours, brown bag lunch, "programming and pizza", book club, Tidy Tuesdays
     * (may be skill specific, e.g. R lunch)
* office hours
* connections with other activities & groups, e.g. research bazaar, code clubs, RLadies, etc.
* invite learners to become Helpers at future workshops, then invite to teach parts of a lesson (before Instructor Training), then invite to apply for Instructor Training
* communications channels (Email lists, Slack, Discord, Teams)
* micro-credentialing

**Other support**
* pointing at challenge banks/other resources for further work 

**Challenges**
* lack of intermediate/continuing material
* shortage of time/effort available for community organizing and tech support
* low attendance/attrition 
* geographic (even across buildings) distribution of trainees

**Comment by dvanic** [comment]
This challenge can be one reason why partnership training with an existing department, lab or other unit work so well - learners end up with a community of learning that they met through the training. In my experience, though, building such relationships works in person and not so much online :(
